Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/SingleZone/VAV/Controller.mo

```diff
@@ -73,7 +73,7 @@ block Controller
     Buildings.Controls.OBC.CDL.Types.SimpleController.PI
     "Type of controller"
     annotation(Dialog(group="Cooling coil control"));
-  parameter Real kCooCoi(final unit="1/K")=0.1
```

Comment:
Should this have unit "1/K"?

Reply:
After adding the unit, when checking the `SingleZone.VAV.Controller` model, I cannot see any warning or error. However, when checking the validation model `SingleZone.VAV.Validation.Controller`, I see following error:
```
Incompatible units.

Incompatible units in
der(conVAV.cooCoi.cooCoiPI.I.y) = conVAV.cooCoi.cooCoiPI.I.k*conVAV.cooCoi.cooCoiPI.I.u;

The unit issue reported appeared in the equation
der(conVAV.cooCoi.cooCoiPI.I.y) = conVAV.cooCoi.cooCoiPI.I.k*conVAV.cooCoi.cooCoiPI.I.u;
Found in class Buildings.Controls.OBC.CDL.Continuous.IntegratorWithReset, /home/jianjun/GitFolder/modelica-buildings/Buildings/Controls/OBC/CDL/Continuous/IntegratorWithReset.mo at line 27, and used in component conVAV.cooCoi.cooCoiPI.I.
```

I did add the unit for the constant source output in commit 3271866. But still see the error.

Reply:
I see, the mistake was in `IntegratorWithReset`, which I fixed in 1fc9b22a840b915fba48a335d5b3d56a8a46e1a6. In that commit, I also added the units back for `kCooCoi`.